src/Uno.UWP/UI/Input/InputCursor.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Microsoft.UI.Input;
+
+#if HAS_UNO_WINUI
+public partial class InputCursor
```

Comment:
Moving InputCursor from Uno.UI.dll to Uno.dll is likely a breaking change. (It should be caught by CI)

I think we shouldn't move it. @jeromelaban could you confirm?

Reply:
cc @dr1rrb

Reply:
Actually, I think the WinUI build in CI will re-generate InputCursor in Uno.UI.dll and mark it with NotImplementedAttribute, and we end up with InputCursor in two assemblies. This might fail the build too due to ambiguity or something.

Reply:
No the CI won't re-generate it. This classes are manually gen'ed on demand, and they are based on UWP contract so far so anyway they won't be generated.

Otherwise yes your right it's a (binary) breaking change :/ The single type that could really cause some problems is the `InputSystemCursor` (the only one that could be manipulated by the apps / third party libs (e.g community toolkit) . I think that we can keep it in the `Uno.UI` project, if so , I would be fine with it. @agneszitte can you confirm if it works?